.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: python
+
+sudo: false
+
+matrix:
+  fast_finish: true
+  include:
+  - python: 3.6
+    env: TEST_TARGET=default
+  - python: 3.6
+    env: TEST_TARGET=coding_standards
+  allow_failures:
+  - python: 3.6
+    env: TEST_TARGET=coding_standards
```

Comment:
This `.travis.yml`  is mostly boilerplate. Please let me know if you have any specific question about it.
I only added to items to the test matrix, a coding standard and a "default" test.